package.json

```diff
@@ -168,6 +168,7 @@
     "ts-expect": "^1.3.0",
     "typescript": "^5.7.3",
     "typescript-eslint": "^8.24.0",
+    "vite": "^6.2.0",
```

Comment:
This is so the Vitest VSCode extension can work.

Reply:
Please don't add it only for that purpose.